canonical/ubuntu.com
# /k8s landing refresh

## Done

- Applied refresh as per [design](https://www.figma.com/file/wGMqsc7u32nWdKQQ9Rv8fP/Ubuntu.com-rebranding?node-id=202%3A408&mode=dev) and [doc](https://docs.google.com/document/d/1b018GoKMF3abEFJlUkfAEK2JEhXWICKF4V0dGdTvgJs/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13638.demos.haus/k8s
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8913
